lib/smack/SmackRep.cpp

```diff
@@ -32,6 +39,9 @@ const Expr* SmackRep::NUL = Expr::id(NULL_VAL);
 const string SmackRep::STATIC_INIT = "$static_init";
 
 const int SmackRep::width = 0;
+const unsigned SmackRep::ptrsize = 32;
```

Comment:
I'm stuck here. To be more specific, if the argument of SmackRep::getElementSize() which used to be SmackRep::getPtrSize is of aggregate type, the reasonable return value should be the size of that type. However, there is no aggregate type in Boogie, which leads me to confusion when I read Boogie code of struct_return.bpl. Any enlightenment?